### PR TITLE
Add 'football-data' API

### LINKED ISSUE #9: Add API

### PR DESCRIPTION
**Pull request by JuKahl**
- ➕ Install dotenv
- 🙈 Add .env
- Add 'football-data' API

In order to hide my authentication token, I have saved it in my .env. However, due to the lack of test possibilities for this variant in Storybook, I am not yet sure whether the corresponding call for the token works correctly.

This may still need to be adjusted.

[API](https://www.football-data.org/documentation/quickstart)

### CODE REVIEW

**Review by DNaunin — APPROVED** [review]
LGTM! ⚽️

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "build-server": "tsc --project tsconfig.server.json"
   },
   "dependencies": {
+    "dotenv": "^6.2.0",
```
Comment (phlgr) [inline package.json]: You don't need to install `dotenv` at all, as this is already built into `next`
https://nextjs.org/docs/basic-features/environment-variables

File: utils/api.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import dotenv from "dotenv";
+
+dotenv.config();
```
Comment (phlgr) [inline utils/api.ts]: https://nextjs.org/docs/basic-features/environment-variables

**Review by phlgr — CHANGES_REQUESTED** [review]

**Review by RonjaMeinecke — APPROVED** [review]
💅🏼GJ

**Pull request merged**